### DISCUSSION TITLE
[Cloud-Engineer] Region Selection

### DISCUSSION THREAD

**Discussion by jawache** [Green-Software-Foundation/patterns#29] [Ideas]
https://docs.aws.amazon.com/wellarchitected/latest/sustainability-pillar/region-selection.html

"Choose Regions near Amazon renewable energy projects and Regions where the grid has a published carbon intensity that is lower than other locations (or Regions)."

- [ ] Testing 1
- [ ] Testing 2
- [ ] Testing 3

**Comment by holanita** [comment]
[Issue created](https://github.com/Green-Software-Foundation/green-software-patterns/issues/74)

**Comment by jawache** [comment]
1. Agenda Item to discuss A

**Reply by jawache** [reply]
My view is this on this agenda item. 
I might not comment but instead I can upvote OR give emojis.

**Comment by jawache** [comment]
2. Agenda Item to diccuss B
